### PR TITLE
make file changes

### PR DESCRIPTION
**Pull request by rahulguptajss**
image

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,71 +1,76 @@
 # Copyright 2021 NetApp, Inc.  All Rights Reserved
+.DEFAULT_GOAL:=help
+SHELL:=/bin/bash
```
Comment (cgrinds) [inline Makefile:3]: Not sure that path will always exist - isn't SHELL := bash sufficient?

**Review by cgrinds — COMMENTED** [review]
Looks good

File: Makefile (hunk 1 above)
Comment (rahulguptajss) [reply Makefile:3]: I think we can remove it all together. Doesn't make any diff

Comment (rahulguptajss) [reply Makefile]: removed it.

**Review by cgrinds — APPROVED** [review]

**Pull request merged**